crm_phonecall_summary_predefined/models/crm_phonecall.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from psycopg2 import IntegrityError
+from openerp import api, fields, models, SUPERUSER_ID
+
+
+class CRMPhonecall(models.Model):
+    _inherit = "crm.phonecall"
+
+    name = fields.Char(
+        related="summary_id.name",
+        store=True,
+        required=False,
+        readonly=True)
+    summary_id = fields.Many2one(
+        comodel_name="crm.phonecall.summary",
+        string="Summary",
+        required=True,
+        ondelete="restrict")
+
+    def _set_default_value_on_column(self, cr, column_name, context=None):
+        """Default values when creating the field."""
+        if column_name != "summary_id":
+            return super(CRMPhonecall, self)._set_default_value_on_column(
+                cr, column_name, context)
+
+        # Ensure crm.phonecall.summary is installed before continuing
+        summary = self.pool["crm.phonecall.summary"]
+        if not summary._table_exist(cr):
+            summary._auto_init(cr, context)
+
+        # Proper default value per row
+        self._init_summary_id(cr, SUPERUSER_ID)
```

Comment:
This should be done as a post_init_hook, not with this hacky technique from old times.